cmd/account/cmd.go

```diff
@@ -38,5 +39,9 @@ func NewCmdAccount(streams genericclioptions.IOStreams, flags *genericclioptions
 }
 
 func help(cmd *cobra.Command, _ []string) {
-	cmd.Help()
+	err := cmd.Help()
+	if err != nil {
+		fmt.Println("Error while calling cmd.Help(): ", err.Error())
+		return
```

Comment:
`return` is not needed as this is the last instruction of the function.
Also this would be consistent with `aao.help` function in which there is no `return` in case of error.

Reply:
This makes sense, I just followed Taylor's advice here to include it